File: gcd_euclidean_algorithm.py
a=int(input("Enter the value of 'a': "))
b=int(input("Enter the value of 'b': "))

rem=1

while rem!=0:
    rem=a%b
    if rem==0:
        print(f'The GCD/HCF (a,b) = {b}')
        break
    a=b
    b=rem

File: gcd_euclidean_algorithm.c
#include<stdio.h>

int main()
{
    int a,b;
    printf("a: ");
    scanf("%i",&a);
    printf("b: ");
    scanf("%i",&b);

    if (a>b)
    {
        int r;
        while (r>=0)
        {
            r=a%b;
            if (r==0)
            {
                printf("gcd(a,b): %i",b);
            }
            a=b;
            b=r;
        }
    }
    else if (b>a)
    {
        int r;
        while (r>=0)
        {
            r=b%a;
            if (r==0)
            {
                printf("gcd(a,b): %i",a);
            }
            b=a;
            a=r;
        }
    }
    else
    {
        printf("gcd(a,b)=%i",a);
    }
    
    return 0;
}